server/e2e/jobs/specs/metadata.e2e-spec.ts

```diff
@@ -33,8 +32,7 @@ describe(`${AssetController.name} (e2e)`, () => {
   });
 
   afterAll(async () => {
-    await db.disconnect();
-    await app.close();
+    await testApp.teardown();
```

Comment:
This is what was causing the process to hang (not cleaning up exiftool-vendored)